CI-API.Data/Repository/AdminRepository.cs

```diff
@@ -855,33 +855,52 @@ public async Task<JsonResult> GetAllMissionApplication(string? search)
 
                 if (search == null)
                 {
-                    List<MissionApplication> missionApplications = cIDbContext.MissionApplications.Where(M => M.ApprovalStatus == StaticCode.missionApplicationPending).ToList();
-                    List<User> userData = cIDbContext.Users.ToList();
-                    List<Mission> missionData = cIDbContext.Missions.ToList();
 
-                    MissionApplicationViewModel allMissionApplications = new()
-                    {
-                        Users = userData,
-                        Applications = missionApplications,
-                        Missions = missionData,
-                    };
+                    var allMissionApplication = await Task.FromResult(cIDbContext.MissionApplications.ToList());
```

Comment:
Remove this line and get allMissionApplication data inside join query below, don't use separate query for that.